client/manager_redis.go

```diff
@@ -0,0 +1,143 @@
+package client
+
+import (
+	"encoding/json"
+
+	"github.com/imdario/mergo"
+	"github.com/ory-am/fosite"
+	"github.com/ory-am/hydra/pkg"
+	"github.com/pborman/uuid"
+	"github.com/pkg/errors"
+	"gopkg.in/redis.v5"
+)
+
+type RedisManager struct {
+	DB        *redis.Client
+	Hasher    fosite.Hasher
+	KeyPrefix string
+}
+
+const redisClientTemplate = "hydra:clients"
```

Comment:
Do you want singular for the table names or plural? Because here it is `hydra:clients` but in the JWK it's singular `hydra:jwk`. Since it's singular in sql/rethink I would recommend having it singular here as well